docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -141,6 +141,57 @@ The following descriptor entry is appended to the descriptor:
 
     * ("generic_key", "<descriptor_value>")
 
+Header Value Match
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: json
+
+  {
+    "type": "header_value_match",
+    "descriptor_value" : "...",
+    "headers" : []
+  }
+
+
+descriptor_value
+    *(required, string)* The value to use in the descriptor entry.
+:ref:`headers<config_http_conn_man_route_table_rate_limit_headers>`
+    *(required, array)* Specifies a set of headers that the rate limit action should match on.
+
+The following descriptor entry is appended to the descriptor if the request matches the headers
+specified in the action config:
+
+    * ("header_match", "<descriptor_value>")
+
+
+.. _config_http_conn_man_route_table_rate_limit_headers:
```

Comment:
I think we now have this text in potentially 3 places in the docs. Can we de-dup this and just link to it from all 3 places?